tasks/pe_uninstall.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Uninstalling PE with the following flags
+# For more information about the uninstaller
+#  and the command-line flags, visit:
+#  https://puppet.com/docs/pe/2019.8/uninstalling.html
+
+sudo ./puppet-enterprise-uninstaller -d -p -y
```

Comment:
Use an absolute path here, otherwise we'll probably not run the command correctly.